### DISCUSSION TITLE
Add a delete button?

### DISCUSSION THREAD

**Discussion by xiaofeiTM233** [vercel/community#701] [Idea]
At _deployments_ page `https://vercel.com/%user%/%project%/deployments` add a **DELETE** button in _Deployment Actions_(three points)
This button can make us deletion faster!
Thanks~

**Comment by swarnava** [comment]
Hi,

I think we kept it hidden on purpose so that people don't accidentally delete their deployment. I have passed your feedback to our internal team to triage this.

**Reply by xiaofeiTM233** [reply]
But you have `Are you sure you want to continue?` 
![20220707162948](https://user-images.githubusercontent.com/55832499/177728751-eaf3ff28-49a6-4d4f-bc41-6e36a56d53b1.png)
They don't accidentally delete their deployment.
And you can turn the `DELETE` red to warn people.

**Reply by swarnava** [reply]
Indeed, we will revisit and see what we can do.
